docs/7.Scan Examples/Git History.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+published: false
+title: SCA scanning
+nav_order: 20
+---
+
+# Git History
+Checkov supports scanning for secrets in git history, to identify and flag secrets which might not be in the current brunches, but are still visible by accessing the git history of the repo.
```

Comment:
```suggestion
Checkov supports scanning of secrets in git history to identify and flag secrets, which might not be in the current branches, but are still visible by accessing the git history of the repo.
```

Reply:
```suggestion
Checkov supports scanning for secrets in git history to identify and flag secrets that might not be in the head commit of the branch but are still visible by accessing the git history of the repo.
```

I want to avoid making it sound like we scan other branches.